tmva/sofie/inc/TMVA/ROperator_Shape.hxx

```diff
@@ -0,0 +1,82 @@
+#ifndef TMVA_SOFIE_ROPERATOR_Shape
+#define TMVA_SOFIE_ROPERATOR_Shape
+
+#include "TMVA/SOFIE_common.hxx"
+#include "TMVA/ROperator.hxx"
+#include "TMVA/RModel.hxx"
+
+#include <sstream>
+#include <iostream>
+#include<sstream>
+#include<vector>
+#include <iterator>
+#include<string>
+namespace TMVA{
+namespace Experimental{
+namespace SOFIE{
+
+template <typename T>
```

Comment:
I think we can remove the template parameter here , since it is not used. The Shape operator works the same on all input types